addons_rksv/pos_rksv/views/res_company.xml

```diff
@@ -23,6 +23,9 @@
                         </group>
                     </page>
                 </notebook>
+                <field name="vat" position="before">
```

Comment:
Probably better after or before ```vat_check_vies``` and we should hide the BMF tab in case it is not checked